llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -300,14 +327,28 @@ def run_event(self, event: Event, state: State, logger: Logger) -> None:
             # Wait for all child processes spawned by the callback to finish.
             timeout = 3600
             wait_start = time.time()
-            while not self._all_child_processes_done():
+            state.device
```

Comment:
typo?

Reply:
oops yes

Reply:
i did that to check something in the IDE
```suggestion
```